### ISSUE TITLE
apex_launchtest still has rclpy dependency

### ISSUE THREAD

**Issue by pbaughman** [ApexAI/apex_rostest#14]
We wanted to get the rclpy dependencies out of apex_launchtest, but it's still a transitive dependency due to ros2launch.

```
root@70699c3fff50:/apex_launchtestt# colcon list --topological-graph --packages-up-to apex_launchtest --packages-above rclpy
rclpy            +**...
launch_ros        +  *.
ros2cli            +**.
ros2pkg             +*.
ros2launch           +*
apex_launchtest       +
```

We depend on ros2launch here: https://github.com/ApexAI/apex_rostest/blob/master/apex_launchtest/apex_launchtest/apex_launchtest_main.py#L21 to get the [print_arguemnts_of_launch_description](https://github.com/ros2/launch/blob/master/ros2launch/ros2launch/api/api.py#L91) function.  This function is also the reason we're not compatible with the ros2 release 'crystal'

Maybe we should go back to a copy/pasted version of this function.

We should add a ci job - maybe scheduled daily or weekly that builds from source following the instructions here: https://index.ros.org/doc/ros2/Installation/Linux-Development-Setup/ to make sure our dependencies are right, and to check to make sure we're not accidentally including rclpy in apex_launchtest

>colcon list --topological-graph --packages-up-to apex_launchtest
>colcon build --packages-up-to apex_launchtest

**Comment by pbaughman** [comment]
I added a ci job that will run on a schedule in [this branch](https://github.com/ApexAI/apex_rostest/blob/ci_isolated_build/.gitlab-ci.yml#L33-L96)  Right now it will fail if you uncomment [this line](https://github.com/ApexAI/apex_rostest/blob/ci_isolated_build/.gitlab-ci.yml#L86)

**Comment by pbaughman** [comment]
If we don't use print_arguments_of_launch_description from launch, we're probably also compatible with the osrf/ros:crystal-desktop docker image, which means we could remove the compatibility blurb about needing a newer version of launch

**Comment by pbaughman** [comment]
@wjwwood Any opinions on how to tackle this?  It's massive overkill to refactor this one function out of ros2launch, right?

**Comment by pbaughman** [comment]
Fixed in #22
